srcpkgs/ncspot/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.11.2
+revision=1
+archs="x86_64"
+wrksrc=${pkgname}-${version}
+create_wrksrc=yes
+#hostmakedepends=""
+#makedepends=""
+depends="libpulseaudio openssl libxcb dbus"
```

Comment:
this should probably be auto-detected if you set makedepends with the proper devel packages